stable/aws-calico/templates/daemon-set.yaml

```diff
@@ -0,0 +1,137 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: calico-node
+  labels:
+    k8s-app: calico-node
+{{ include "aws-calico.labels" . | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+      k8s-app: calico-node
```

Comment:
`k8s-app` is not a standard label, please use `app.kubernetes.io` labels everywhere, see https://github.com/aws/eks-charts/blob/master/stable/appmesh-controller/templates/deployment.yaml#L11

Reply:
changed